snap/snapcraft.yaml

```diff
@@ -131,8 +131,8 @@ parts:
       - mysql-server-8.0=8.0.36-0ubuntu0.22.04.1
       - mysql-router=8.0.36-0ubuntu0.22.04.1
       - mysql-shell=8.0.36+dfsg-0ubuntu0.22.04.1~ppa4
-      - prometheus-mysqld-exporter=0.14.0-0ubuntu0.22.04.1~ppa1
-      - prometheus-mysqlrouter-exporter=4.0.5-0ubuntu0.22.04.1~ppa1
+      - prometheus-mysqld-exporter=0.14.0-0ubuntu0.22.04.1~ppa2
+      - prometheus-mysqlrouter-exporter=5.0.1-0ubuntu0.22.04.1~ppa1
```

Comment:
just to be sure, this is the version with the fix with graceful termination?

Reply:
yes, exactly. [5.0.1](https://github.com/rluisr/mysqlrouter_exporter/blob/v5.0.1/main.go#L376) has graceful termination